build-system/pr-check/bundle-size.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview
+ * This script downloads the module and nomodule builds then runs the bundle
+ * size check.
+ * This is run during the CI stage = bundle-size; job = Bundle Size.
+ */
+
+const colors = require('ansi-colors');
+const {
+  downloadEsmDistOutput,
+  downloadDistOutput,
+  printChangeSummary,
+  startTimer,
+  stopTimer,
+  stopTimedJob,
+  timedExecOrDie: timedExecOrDieBase,
+} = require('./utils');
+const {determineBuildTargets} = require('./build-targets');
+const {isTravisPullRequestBuild} = require('../common/travis');
+const {runNpmChecks} = require('./npm-checks');
+
+const FILENAME = 'bundle-size.js';
+const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
+const timedExecOrDie = (cmd) => timedExecOrDieBase(cmd, FILENAME);
+
+async function main() {
+  const startTime = startTimer(FILENAME, FILENAME);
+  if (!runNpmChecks(FILENAME)) {
+    stopTimedJob(FILENAME, startTime);
+    return;
+  }
+
+  if (!isTravisPullRequestBuild()) {
+    downloadDistOutput(FILENAME);
+    downloadEsmDistOutput(FILENAME);
+    timedExecOrDie('gulp bundle-size --on_push_build');
+  } else {
+    printChangeSummary(FILENAME);
+    const buildTargets = determineBuildTargets(FILENAME);
+    if (buildTargets.has('RUNTIME') || buildTargets.has('FLAG_CONFIG')) {
+      downloadDistOutput(FILENAME);
+      downloadEsmDistOutput(FILENAME);
+      timedExecOrDie('gulp bundle-size --on_skipped_build');
+    } else {
+      console./*OK*/ log(
```

Comment:
Missing `gulp bundle-size --on_skipped_build` call

Reply:
done. and added `await signalDistUpload('skipped');` right after